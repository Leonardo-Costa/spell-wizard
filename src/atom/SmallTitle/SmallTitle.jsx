import { StyleSheet, Text, View } from "react-native";
import colors from "../../misc/Colors";
import React from "react";

function SmallTitle({ children, style }) {
  return (
    <View style={[styles.container, style]}>
      <Text style={styles.title}>{children}</Text>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {},
  title: {
    fontWeight: "bold",
    fontSize: 25,
    color: colors.dark_gray,
  },
});

export { SmallTitle };
