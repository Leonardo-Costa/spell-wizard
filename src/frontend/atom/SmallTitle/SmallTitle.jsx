import { StyleSheet, Text, View } from 'react-native';
import React from 'react';
import colors from '../../misc/Colors';

const styles = StyleSheet.create({
  container: {

  },
  title: {
    fontWeight: 'bold',
    fontSize: 25,
    color: colors.dark_gray,
  },
});

function SmallTitle({ text }) {
  return (
    <View style={styles.container}>
      <Text style={styles.title}>{text}</Text>
    </View>
  );
}

export default { SmallTitle };
