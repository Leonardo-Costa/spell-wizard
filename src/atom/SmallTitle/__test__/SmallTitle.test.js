import React from 'react';
import renderer from 'react-test-renderer';

import {SmallTitle} from '../SmallTitle';

it('renders correctly', () => {
    const tree = renderer.create(<SmallTitle />).toJSON();
    expect(tree).toMatchSnapshot();
  });