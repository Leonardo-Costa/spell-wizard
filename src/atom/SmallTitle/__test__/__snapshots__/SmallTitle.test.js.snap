// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {},
      undefined,
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#202020",
          "fontSize": 25,
          "fontWeight": "bold",
        },
        undefined,
      ]
    }
  />
</View>
`;
